docs/diagrams/ImportSequenceDiagram.puml

```diff
@@ -0,0 +1,49 @@
+@startuml
+!include style.puml
+
+mainframe **Import** implementation
+
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":ImportCommand" as ImportCommand LOGIC_COLOR
+participant ":CommandResult" as CommandResult LOGIC_COLOR
+
+box Storage STORAGE_COLOR_T1
+participant "jfm:JsonFileManager" as JsonFileManager STORAGE_COLOR
+box
+
+create ImportCommand
+LogicManager -> ImportCommand : parser.parseCommand(commandText)
+activate ImportCommand
+
+ImportCommand --> LogicManager
+deactivate ImportCommand
+
+LogicManager -> ImportCommand : execute(model)
+activate ImportCommand
+
+create JsonFileManager
+ImportCommand -> JsonFileManager : new JsonFileManager()
+activate JsonFileManager
+
+JsonFileManager --> ImportCommand :
+deactivate JsonFileManager
+
+ImportCommand -> JsonFileManager : importFile(model.getShowListFilePath)
+activate JsonFileManager
+
+JsonFileManager --> ImportCommand : result
+deactivate JsonFileManager
+
+create CommandResult
+ImportCommand -> CommandResult : new CommandResult(MESSAGE_SUCCESS, false, false, true)
+activate CommandResult
+
+CommandResult --> ImportCommand
+deactivate CommandResult
+
+ImportCommand --> LogicManager : commandResult
```

Comment:
That's true, since the Import and Export sequence diagrams are almost the same except for the ImportCommand and ExportCommand classes, I thought I could just leave them out.
It would be more beneficial for the user to place them into the DG though.